snapshots/snapshot_generator/src/test/resources/expected-snapshot-images.txt

```diff
@@ -0,0 +1,5 @@
+{"locations":[{"linkText":"Link text: HwTmMGll","url":"https://iiif.wellcomecollection.org/image/RxS.jpg/info.json","credit":"Credit line: lt4dBhn","license":{"id":"cc-by","label":"Attribution 4.0 International (CC BY 4.0)","type":"License","url":"http://creativecommons.org/licenses/by/4.0/"},"accessConditions":[],"locationType":{"id":"iiif-image","label":"IIIF Image API","type":"LocationType"},"type":"DigitalLocation"}],"source":{"subjects":[],"id":"n198nbcy","languages":[],"genres":[],"title":"title-FUXMuZ6vHG","type":"Work","contributors":[]},"aspectRatio":0.8347602,"thumbnail":{"linkText":"Link text: HwTmMGll","url":"https://iiif.wellcomecollection.org/image/RxS.jpg/info.json","credit":"Credit line: lt4dBhn","license":{"id":"cc-by","label":"Attribution 4.0 International (CC BY 4.0)","type":"License","url":"http://creativecommons.org/licenses/by/4.0/"},"accessConditions":[],"locationType":{"id":"iiif-image","label":"IIIF Image API","type":"LocationType"},"type":"DigitalLocation"},"averageColor":"#458C97","id":"pquzghlq","type":"Image"}
```

Comment:
Wasn't really sure how to do this - am not especially au fait with the test documents stuff cc @alexwlchan